src/averageAges.js

```diff
@@ -37,7 +52,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let womenList = people.filter(person => person.sex === 'f');
+  const mothers = [];
+
+  people.map(person => mothers.push(person.mother));
```

Comment:
It is possible to make it shorter
```suggestion
  const mothers = people.map(person => person.mother);
```